routers/web/repo/release.go

```diff
@@ -330,9 +330,9 @@ func newReleaseCommon(ctx *context.Context) {
 	ctx.Data["Title"] = ctx.Tr("repo.release.new_release")
 	ctx.Data["PageIsReleaseList"] = true
 
-	tags, err := repo_model.GetTagNamesByRepoID(ctx, ctx.Repo.Repository.ID)
+	tags, err := repo_model.GetTagMappingsByRepoID(ctx, ctx.Repo.Repository.ID)
```

Comment:
I do not think it is right because `GetTagNamesByRepoID` does `order by`, but `map` is unordered,